Source/Core/DolphinQt/Config/WiimoteControllersWidget.cpp

```diff
@@ -123,6 +126,26 @@ void WiimoteControllersWidget::CreateLayout()
   // Passthrough BT
   m_wiimote_layout->addWidget(m_wiimote_passthrough, m_wiimote_layout->rowCount(), 0, 1, -1);
 
+  int adapter_row = m_wiimote_layout->rowCount();
+  m_wiimote_layout->addWidget(m_bluetooth_adapters_label, adapter_row, 1, 1, 1);
+  m_wiimote_layout->addWidget(m_bluetooth_adapters, adapter_row, 2, 1, 2);
+  const int selected_vid = Config::Get(Config::MAIN_BLUETOOTH_PASSTHROUGH_VID);
+  const int selected_pid = Config::Get(Config::MAIN_BLUETOOTH_PASSTHROUGH_PID);
+  for (auto device : IOS::HLE::BluetoothRealDevice::ListDevices())
+  {
+    std::string name = device.name.empty() ? tr("Unknown Device").toStdString() : device.name;
+
+    QString device_info =
+        QString::fromStdString(fmt::format("{} ({:04x}:{:04x})", name, device.vid, device.pid));
+
+    m_bluetooth_adapters->addItem(device_info);
+    m_bluetooth_adapters->setItemData(m_bluetooth_adapters->count() - 1,
+                                      QVariant::fromValue(device));
+
+    if (device.vid == selected_vid && device.pid == selected_pid)
```

Comment:
If the explicitly configured device is not currently attached, dolphin will not show the correct value. Maybe do something like the "Device" list in the controller config window, if the configured device is not present a `[disconnected] VID:PID` entry is added, just so the user can see what's going on better.